_layouts/default.html

```diff
@@ -20,18 +21,17 @@
 
 	<nav id="nav">
 		<div class="container">
-			<div class="row">
-				<div class="col-xs-3">
-					<a href="/" class="logo">The Lounge</a>
-				</div>
-				<div class="col-xs-9 right">
-					<a href="/docs">Documentation</a>
-					<a href="https://github.com/thelounge/lounge" class="github">
-						GitHub
-					</a>
-					<!--<a href="/blog">Blog</a>-->
-				</div>
-			</div>
+			<a href="/" class="logo">
+				<img alt="The Lounge" src="/assets/logos/logo/TL_White&Yellow_Horizontal_logotype_Transparent_Bg/TL_White&Yellow_Horizontal_logotype_Transparent_Bg.svg">
+			</a>
+
+			<ul class="nav-links">
+				<li><a href="/docs">Docs</a></li>
```

Comment:
I think we should keep the full `Documentation`.

Reply:
Fair enough, done.